project-base/app/tests/App/Functional/Model/Order/OrderFacadeEditTest.php

```diff
@@ -102,6 +102,10 @@ public function testEditProductItemWithoutUsingPriceCalculation(): void
 
     public function testAddProductItem(): void
     {
+        $this->markTestSkipped('Adding new items into Order is denied. It is caused by unknown ProductType for new order items.'
+            . ' If you need it, It can be solved by filling OrderItemData and calling new methods for creating OrderItems');
+
+        // @phpstan-ignore-next-line Tests are skipped
```

Comment:
This could mean considerable changes in the already introduced adjustments.
I would like to solve this in a separate task, since this one is already quite large.